braket_features/Getting devices and checking device properties.ipynb

```diff
@@ -0,0 +1,340 @@
+     ]
+    }
+   ],
+   "source": [
+    "# the IonQ device\n",
+    "device = AwsDevice(\"arn:aws:braket:::device/qpu/ionq/ionQdevice\")\n",
+    "\n",
+    "# the Rigetti device\n",
+    "device = AwsDevice(\"arn:aws:braket:::device/qpu/rigetti/Aspen-8\")\n",
+    "\n",
+    "execution_windows = device.properties.service.executionWindows[0]\n",
```

Comment:
I'd remove [0] here. If the list were longer then we'd only be returning the first window.

Reply:
No, this is just to get the element out of a nested format, no useful information is lost here.

Reply:
Why is executionWindows a list if there's always only one element?

Reply:
It is a nested list. I just remove the nest.

Reply:
It doesn't look like a nested list to me? The output you had for executionWindows[0] is
executionDay=<ExecutionDay.EVERYDAY: 'Everyday'> windowStartHour=datetime.time(15, 0) windowEndHour=datetime.time(19, 0)
which does not look like a list.